tests/mgbench/runners.py

```diff
@@ -79,8 +78,16 @@ def _get_args(self, **kwargs):
         if self._memgraph_version >= (0, 50, 0):
             kwargs["storage_properties_on_edges"] = self._properties_on_edges
         else:
-            assert self._properties_on_edges, \
-                "Older versions of Memgraph can't disable properties on edges!"
+            assert self._properties_on_edges, "Older versions of Memgraph can't disable properties on edges!"
+
+        if self._extra_args != "":
+            args_list = self._extra_args.split(" ")
+            assert len(args_list) % 2 == 0
+            for i in range(0, len(args_list), 2):
+                kwargs[args_list[i]] = args_list[i + 1]
+
+        kwargs["split-file"] = self._split_file
```

Comment:
This doesn't work with regular Memgraph (built on `master`), I would make the `mgbench` code totally independent of what "version" of Memgraph is run so that we can easily merge `mgbench` code from master or the other way around + we don't have to change the code later on. In other words, remove the split files logic from `tests/mgbench` and use extra flags to pass that info to Memgraph.